package.json

```diff
@@ -128,7 +128,7 @@
     "load-grunt-config": "0.19.2",
     "marked": "^0.8.0",
     "mini-css-extract-plugin": "^0.4.4",
-    "ng-packagr": "5.5.0",
+    "ng-packagr": "^9.1.5",
```

Comment:
Just want to verify we weren't able to use v10 of ng-packagr? I know we avoided v9 when we updated to Angular 9 because we got build warnings:

![image](https://user-images.githubusercontent.com/20795331/87936611-0c121d00-ca8b-11ea-95db-211758512c8d.png)

Due to an issue where it didn't remove interface imports from the generated bundle:
https://github.com/ng-packagr/ng-packagr/issues/1559

Its harmless as tree-shaking will remove them as unused imports, but it was supposedly fixed in v10 of ng-packagr, just wanted to check if there were issues using v10

Reply:
Unfortunately I forgot where I saw the "Class constructor x cannot be invoked without 'new'" error. But according to this we lose our es5 bundle when we upgrade to ng-packagr@10:

https://github.com/angular/angular-cli/issues/18067#issuecomment-651647074

Do we still need that for IE11 support? At least for apps which don't use Angular CLI, like our documentation?

Reply:
From my understanding (I could be wrong) but we don't need to ship an es5 bundle any more regardless of whether we need to support IE11. It was primarily for the bundling tools (e.g. webpack, rollup, systemjs) as they select the format that best suits them. In the case of CLI apps that rely on Webpack, as far as I know they will all be using the esm2015/fesm2015 bundles and have been for quite some time.

Essentially, the downleveling to ES5 happens as part of the application build rather than when we build our library so it removes the need for these other formats.

However if you are getting another issue anyway then perhaps we can't update it anyway.

Reply:
Ah I just saw the issue link you posted in which case there could be an issue as application downleveling was a change in the Angular 10 CLI, so just ignore this point, we can just ignore the warnings

Reply:
Okay that's good to know. Let me try to reproduce the issue, there might be another workaround for it that allows us to upgrade properly.

Reply:
Based on this comment in the issue you linked:

![image](https://user-images.githubusercontent.com/20795331/87950451-35d53f00-ca9f-11ea-9f84-c9b16c339e1c.png)

Looks like we will need to stick with ng-packagr v9, as application downleveling was only added in v10 so we will still need to provide es5 bundles 🙄 my misunderstanding, sorry.

Reply:
Thanks for confirming, then I guess we are stuck with this version until Angular 12 comes out.

Reply:
Looks like it! That'll be fine, when v11 comes out we can perform Ivy builds using ng-packagr 9 so that will be an improvement we can make without having to upgrade before v12.